### PR TITLE
Minor grammatical editing

### PR DESCRIPTION
**Pull request by szabogabor91**
Some lines were technically not incorrect grammatically, but used rarely used spelling.

### CODE REVIEW

File: src/concepts/the-sane-helm-guide.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ But Helm does two things well:
 - packaging
 - and templating
 
-In this guide you will get practical knowledge on how-to use Helm focusing on those usecases, packaging and templating, that by the way also underpin Gimlet.
+In this guide, you will get practical knowledge on how to use Helm, focusing on those use-cases, packaging and templating, that, by the way, also underpin Gimlet.
```
Comment (laszlocph) [inline src/concepts/the-sane-helm-guide.md:17]: I've been fighting with this sentence for a while. It also popped up on your radar. Good.

How about we remove the `, by the way,` completely?

File: src/concepts/the-sane-helm-guide.md [hunk 2]
```diff
@@ -222,7 +222,7 @@ metadata:
   [...]
 ```
 
-The output is well known - although lengthy - Kubernetes yaml. These are the yamls Helm is applying on the cluster, this time though it was only rendering text files on your laptop.
+The output is a well known - although lengthy - Kubernetes yaml. These are the yamls Helm is applying on the cluster, this time though it was only rendering text files on your laptop.
```
Comment (laszlocph) [inline src/concepts/the-sane-helm-guide.md:225]: Using `the` instead of `a`? Otherwise this needs fix, agreed.

**Review by laszlocph — COMMENTED** [review]
Two recommendations, otherwise thanks for fixing these. Good recommendations!

File: src/concepts/the-sane-helm-guide.md (hunk 2 above)
Comment (szabogabor91) [reply src/concepts/the-sane-helm-guide.md:225]: Sure, "the" can work just as well.

File: src/concepts/the-sane-helm-guide.md (hunk 1 above)
Comment (szabogabor91) [reply src/concepts/the-sane-helm-guide.md:17]: Thought about this one for a little bit. How about something like this?

"In this guide, you will get practical knowledge about using Helm, focusing on those use-cases, namely packaging and templating, that also underpin Gimlet."

**Comment by laszlocph** [comment]
Please make the adjustments @szabogabor91 so I can merge.
I have only cumbersome workflows to make the changes: a follow up pr, or merge then commit to master. You have the branch and make the small change. Thanks.

**Comment by szabogabor91** [comment]
> Please make the adjustments @szabogabor91 so I can merge.
> I have only cumbersome workflows to make the changes: a follow up pr, or merge then commit to master. You have the branch and make the small change. Thanks.

I have made the requested changes. Could you please double check them?

**Pull request merged**